MailCore/Cache/Attachments/AttachmentsManagerWorker/AttachmentsManagerWorker.swift

```diff
@@ -370,22 +370,14 @@ extension AttachmentsManagerWorker: AttachmentsManagerWorkable {
         await updateDelegate?.contentWillChange()
     }
 
-    public func processTextAttachments(_ attachments: [TextAttachable]) async {
-        // Process all text attachments
-        let textAttachments = await attachments.concurrentMap { attachment in
-            await attachment.textAttachment
-        }
-
+    public func processHtmlAttachments(_ htmlAttachments: [HTMLAttachable]) async {
```

Comment:
I would call the function processHTMLAttachments here too (as it would follow Swift naming rules)